docs/README.md

```diff
@@ -0,0 +1,14 @@
+# ROS build farm deployment
```

Comment:
This file is not linked from the main `README.md`, but the other `*.md` files in the `/doc` directory are, so it is quite possible that this file will never be read. 

Also, it seems to talk about deployment. So maybe we could rename it to 'deployment.md` which, by the way, is actually linked in the main `README.md`?